docs/man/ctags.1.rst

```diff
@@ -491,7 +491,7 @@ are not listed here. They are experimental or for debugging purpose.
 		ctags generates an anonymous extra tag for the struct
 		and fills the scope fields with the name of the extra tag.
 
-		.. code-block::
+		.. code-block:: ctags
```

Comment:
docs/man/* files are generated from man/* files.
So you must fix the original file under man/.
After fixing the files under man/, don't forget to run 
```
$ make QUICK=1 update-docs
```
at man/ directory.  "make" command may generate docs/man/* files.
You must add the changes for docs/man/* to your commit.

Reply:
fixed.